src/approaches/interactive_learning_approach.py

```diff
@@ -2,7 +2,7 @@
 """
 
 import itertools
```

Comment:
Are the changes in this file and test_interactive_learning_approach hand-in-hand with the PlayroomEnv? If not, we could separate out the PRs

Reply:
hm these changes should already be in master -- i thought i had removed these commits from this branch but i guess not, will fix that